### DISCUSSION TITLE
forward group A's chat to bot B

### DISCUSSION THREAD

**Discussion by ghost** [MrMissx/Telegram_Forwarder#32] [Q&A]
Hello, first of all, thanks for a good script. I have a question. I want to forward group A's chat to bot B. But I don't know how to set it up. @bot_id doesn't work, chat id doesn't seem to work either. Help!

**Comment by MrMissx** [comment]
Currently a bot can't comunicate with other bot (eg: sending, reading messages) due to Telegram limitation. So it won't work anyway.
You can consider to use userbot if you want to do that.
